lib/pages/timetable/model/events/uni_event.dart

```diff
@@ -158,14 +159,16 @@ class UniEventInstance extends Event {
   final Color color;
   final String location;
   final String info;
+  final bool active;
```

Comment:
What's the point of this?

Reply:
Will be used to verify if the instance of the event is active or canceled